### PR TITLE
add commnets endpoint and can go to reviews from categories as well

### PR DESCRIPTION
**Pull request by AwaisQarni4**
Can see comments and as per Alex review now I can see reviews via tha Categories endpoint too, no need to go back. 

. The fetch.js file I have left for refactoring at the minutes will definitely do after doing the main tasks

### CODE REVIEW

File: src/components/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { useParams } from "react-router-dom";
```
Comment (jimstev2447) [inline src/components/Comments.jsx:1]: Is it intentional that you only want to show the comments on their own and not with the review?

File: src/components/ShowComments.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const ShowComments = ({ comment }) => {
+  return (
+    <div>
```
Comment (jimstev2447) [inline src/components/ShowComments.jsx]: Should this be a div, please be mindful of semantic html

File: src/components/Comments.jsx (hunk 1 above)
Comment (AwaisQarni4) [reply src/components/Comments.jsx:1]: yes, it was intentional, can change it tho

File: src/components/ShowComments.jsx (hunk 2 above)
Comment (AwaisQarni4) [reply src/components/ShowComments.jsx]: oh right, will change accordingly

**Comment by jimstev2447** [comment]
One thing, I can't actually navigate to anywhere on your app once I have viewed the comments, It might be nice to be able to navigate back to the article or to the home page

**Pull request merged**